packages/pre/src/porter.ts

```diff
@@ -0,0 +1,17 @@
+import { getPorterUri as doGetPorterUri } from '@nucypher/shared';
+
+type PreDomain = 'oryx';
```

Comment:
This domain is planned to be EOL after the next release, Shall we update it? 

```suggestion
type PreDomain = PRE_DOMAIN.TESTNET;
```

Reply:
We can update it now, sure. I gather from comments in this PR that it's supposed to be `tapir`?

WRT to the line: Notice how `type PreDomain = 'oryx';` is used in `PRE_DOMAIN` type definition:

```
export const PRE_DOMAIN: Record<string, PreDomain> = {
  TESTNET: 'oryx',
};
```

It works here like an enum - It describes the legal values in `PRE_DOMAIN` dictionary.